modules/logging-release-notes-5-9-1.adoc

```diff
@@ -0,0 +1,35 @@
+//module included in logging-5-9-release-notes.adoc
+:content-type: REFERENCE
+[id="logging-release-notes-5-9-1_{context}"]
+= Logging 5.9.1
+This release includes link:https://access.redhat.com/errata/RHSA-2024:2096[OpenShift Logging Bug Fix Release 5.9.1]
+
+[id="logging-release-notes-5-9-1-bug-fixes"]
+== Bug Fixes
+
+* Before this update, a bug in LogQL parsing left out some line filters from the query. With this update, the parsing now includes all the line filters while keeping the original query unchanged. (link:https://issues.redhat.com/browse/LOG-5268[LOG-5268])
+
+* Before this update, a prune filter without a defined `pruneFilterSpec` would cause a segfault. With this update, there is a validation error if a prune filter is without a defined `puneFilterSpec`. (link:https://issues.redhat.com/browse/LOG-5322[LOG-5322])
+
+* Before this update, a drop filter without a defined `dropTestsSpec` would cause a segfault. With this update, there is a validation error if a prune filter is without a defined `puneFilterSpec`. (link:https://issues.redhat.com/browse/LOG-5323[LOG-5323])
+
+* Before this update, the {loki-op} did not validate the S3 endpoint used in the storage secret. With this update, the validation process ensures the S3 endpoint is a valid S3 URL, and the `LokiStack` status updates to indicate any invalid URLs. (link:https://issues.redhat.com/browse/LOG-5395[LOG-5395])
+
+* Before this update, the {loki-op} did not validate the correctness of the S3 endpoint URL format used in the storage secret. With this update, the S3 endpoint URL goes through a validation step that reflects on the status of the `LokiStack`. (link:https://issues.redhat.com/browse/LOG-5397[LOG-5397])
+
+* Before this update, {loki-op} set up Loki to use path-based style access for AWS S3, which has been deprecated. With this update, it defaults to virtual-host style without users needing to change their configuration. (link:https://issues.redhat.com/browse/LOG-5401[LOG-5401])
+
+* Before this update, poorly formatted timestamp fields in audit log records led to `WARN` messages in {clo} logs. With this update, a remap transformation ensures that the timestamp field will be properly formatted. (link:https://issues.redhat.com/browse/LOG-4672[LOG-4672])
+
+* Before this update, the validation feature for {logging} didn't align with the current documentation. With this update, the validation feature for {logging} is aligned with the documentation. Now, during validation, the system checks if a `ClusterLogForwarder` with the same name exists in the same namespace. If not, it throws an exception. (link:https://issues.redhat.com/browse/LOG-5062[LOG-5062])
```

Comment:
> did not align with the current documentation
This doesn't seem right. The code doesn't align with the docs; the doc align with the code. I am not sure how to correct this, as I don't fully understand the issue. 

Also, the bug Jira is `Unresolved`. Should this one be documented?

Reply:
It's marked as `Done-Errata` now. So documented and rephrased to make more sense. Thanks!